python/cross_service/textract_explorer/textract_app.py

```diff
@@ -9,12 +9,12 @@
 image.
 """
 
-from io import BytesIO
 import logging
-import tkinter
 import threading
+import tkinter
+from io import BytesIO
+
 from PIL import Image, ImageTk
-from textract_wrapper import TextractWrapper
```

Comment:
Is this import not needed?

Reply:
Correct. It's passed in during class instantiation during the demo, as a class param.